scripts/build-wheels.sh

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env bash
 set -e
 pip install setuptools wheel
-cd semgrep && python setup.py sdist bdist_wheel
+cd semgrep && SEMGREP_SKIP_BIN=true python setup.py sdist bdist_wheel
```

Comment:
I don't think we want to do this here, we need the binaries when building the wheel.